Makefile

```diff
@@ -262,32 +257,31 @@ build-tests:
 
 unit-test: clean-test-results
 	@printf $(COLOR) "Run unit tests..."
-	@go test $(UNIT_TEST_DIRS) -timeout=$(TEST_TIMEOUT) $(TEST_TAG) -race | tee -a test.log
+	@go test $(TEST_DIRS) -timeout=$(TEST_TIMEOUT) -tags=unittest -race | tee -a test.log
```

Comment:
Why did you add `unittest` build tag? I don't like the idea of mixing dir and tag based approaches.

Reply:
What about I change everything to use build tag?

Reply:
I remove the unit test build tag and still use dir to filter tests. The problem is golang only support negative filtering. If I want to use the tag to filter integration test, I need to add `!intergrationtest` tag to all unit tests. I am not sure if there is other way around.